### DISCUSSION TITLE
Ability to return to full timeline view from an album

### DISCUSSION THREAD

**Discussion by bonobo-d** [ente-io/ente#697] [Enhancements]
This is a feature request for the desktop app (I'm on linux).

I want to open an album, then be able to easily go back to the full timeline view where all pictures are shown. Currently I know of two ways to do this:

- Find the "All" album in the top gallery and click on it. This is very cumbersome when you have lots of albums, because sometimes I have to click on the little "left" arrow in the album list many times before I can get to "All".
- Hit Ctrl+R to refresh the app. This feels wrong.

I would expect the back keybindings in my keyboard and mouse to take me back to the previous view. Also, a simple "back to timeline" button would serve the purpose.

**Comment by mnvr** [comment]
Duplicate of https://github.com/ente-io/ente/discussions/530

> Use navigator history to navigate between albums

but maybe this is phrased better so will keep this one and close the other.

**Comment by RolfSievert** [comment]
Being able to use the "navigate back" and "navigate forward" buttons on the mouse would be great. Just like it works in the browser.
